solace-spring-boot-autoconfigure/solace-java-spring-boot-autoconfigure/pom.xml

```diff
@@ -34,6 +34,12 @@
 			<version>${solace.jcsmp.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-oauth2-client</artifactId>
+		</dependency>
+
+
```

Comment:
Correct me if I'm wrong, but isn't OAuth2 usually an opt-in dependency in Spring? Maybe this should be optional here as well?

Reply:
yes this optional dependency. I'm not 100% sure how optional actually works. I will read more on optional and will make it optional.